package.json

```diff
@@ -46,7 +46,7 @@
   ],
   "dependencies": {
     "ember-cli-babel": "^5.1.7",
-    "ember-macro-helpers": "0.3.0"
+    "ember-macro-helpers": "kellyselden/ember-macro-helpers#set-resolve"
```

Comment:
I will officially release the changes required for this PR if the concept is approved by you.